app/lib/isAuth.tsx

```diff
@@ -0,0 +1,37 @@
+const clientId = process.env.NEXT_PUBLIC_CTP_CLIENT_ID;
+const clientSecret = process.env.NEXT_PUBLIC_CTP_CLIENT_SECRET;
+const authUrl = process.env.NEXT_PUBLIC_CTP_AUTH_URL;
```

Comment:
Same problem. Do not use .env in each file

Reply:
I've created a new file utils/constants.ts to store environment variables and endpoint constants related to the commercetools platform.